packages/angular_devkit/build_angular/src/builders/extract-i18n/options.ts

```diff
@@ -33,8 +33,16 @@ export async function normalizeOptions(
   const projectMetadata = await context.getProjectMetadata(projectName);
   const projectRoot = path.join(workspaceRoot, (projectMetadata.root as string | undefined) ?? '');
 
-  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-  const buildTarget = targetFromTargetString(options.buildTarget ?? options.browserTarget!);
+  // Target specifier defaults to the current project's build target with no specified configuration
+  const buildTargetSpecifier = options.buildTarget ?? options.browserTarget ?? `:build`;
```

Comment:
Why not provide the default in the schema so that it will also be documented?

Reply:
It breaks the handling of the `builderTarget` and deprecated `browserTarget` handling. `builderTarget` will always have a value causing `browserTarget` to never be used. Once we remove `browserTarget` we can definitely do that.